model-optimizer/mo/utils/version.py

```diff
@@ -14,12 +14,34 @@
  limitations under the License.
 """
 import os
+import subprocess
+
+
+def get_version_file_path():
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir, "version.txt")
+
+
+def generate_mo_version():
+    """
+    Function generates version like in cmake
+    custom_{branch_name}_{commit_hash}
+    """
+    branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip().decode()
+    commit_hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode()
+    version = "custom_{}_{}".format(branch_name, commit_hash)
+    with open(get_version_file_path(), 'w') as f:
+        f.write(version)
+    return version
 
 
 def get_version():
-    version_txt = os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir, "version.txt")
+    version_txt = get_version_file_path()
     if not os.path.isfile(version_txt):
-        return "unknown version"
+        return generate_mo_version()
     with open(version_txt) as f:
         version = f.readline().replace('\n', '')
     return version
+
+
+def get_mo_version():
+    return get_version()
```

Comment:
why do we need `get_mo_version` function?

Reply:
It was intersecting with imported get_versoin() function from IE. But I resolved so `get_mo_version` is no more needed.